.github/CODEOWNERS

```
@@ -0,0 +1,26 @@
+# anything with no explicit code owners will be tagged to @core-devs
+*   @napari/core-devs
+
+# submodules
+napari/_vispy/      @brisvag @melonora
+napari/_qt/         @Czaki @lucyleeow @DragaDoncila
```

```suggestion
napari/_qt/         @Czaki @lucyleeow @DragaDoncila @psobolewskiPhD 
```
Could salami slice out the qpp and plugin parts, but this is fine I think.